Alexandria/.classpath

```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file needs to be removed from the branch, by including .classpath in the gitignore